tardis/apps/publication_workflow/components/PublicationCard.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Fragment } from 'react';
+import styled from '@emotion/styled';
+import { Button } from 'react-bootstrap';
+import moment from 'moment';
+
+const Hover = styled.card({
+  ':hover': {
+    transform: 'scale(1.01)',
+    boxShadow: '0 10px 20px rgba(0,0,0,.12), 0 4px 8px rgba(0,0,0,.06)',
+  },
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
    },
```